src/components/app/app.js

```diff
@@ -0,0 +1,62 @@
+import MainTable from '../main-table/main-table';
+import EditPanel from '../edit-panel/edit-panel';
+import Header from '../header/header';
+import SelectDeck from '../select-deck/select-deck';
+import CreateDeck from '../create-deck/create-deck';
+import { Component } from 'react';
+import { BrowserRouter as Router, Route, Switch } from 'react-router-dom'; 
+
+import './app.css';
+import WelcomePage from '../welcome-page/welcome-page';
+
+const store = require('store');
+
+class App extends Component {
+  constructor (props) {
+    super(props);
+    this.state = {
+      deck: [
+        {eng: "Money", rus: "Деньги"},
+        {eng: "Tiger", rus: "Тигр"},
+        {eng: "Pen", rus: "Ручка"},
+        {eng: "Bottle", rus: "Бутылка"},
+        {eng: "Tree", rus: "Дерево"},
+      ]
+    }
+  }
+
+  setItemsToLocalStorage = () => {
```

Comment:
please avoid set business logic to the root app file, instead move it to the appropriate component